tests/unit/nginx-vhosts_6.bats

```diff
@@ -107,6 +107,44 @@ teardown() {
   assert_failure
 }
 
+@test "(nginx-vhosts) nginx:set --global hsts" {
+  setup_test_tls wildcard
+  local HSTS_CONF="/home/dokku/${TEST_APP}/nginx.conf.d/hsts.conf"
+
+  run deploy_app
+  echo "output: $output"
+  echo "status: $status"
+  assert_success
+  assert_output_contains "Enabling HSTS"
+
+  # disable hsts globally
+  run /bin/bash -c "dokku nginx:set --global hsts false"
+  # check it is now not applied
+  run /bin/bash -c "dokku nginx:build-config $TEST_APP"
+  echo "output: $output"
+  echo "status: $status"
+  assert_success
+  assert_output_contains "Enabling HSTS" 0
+
+  # apply on app
+  run /bin/bash -c "dokku nginx:set $TEST_APP hsts true"
+  # check it is now applied
+  run /bin/bash -c "dokku nginx:build-config $TEST_APP"
+  echo "output: $output"
+  echo "status: $status"
+  assert_success
+  assert_output_contains "Enabling HSTS" 1
+
+  # disable globally
```

Comment:
Should say `implicit enable globally`.

Should we also have an assert for implicitly enabling?

Reply:
hm first need to get the test working... then will look at code comments... the non-ability to just run one test-case without waiting an hour is just enough to make me think about walking away.. I think the code works (works in my env; cat>./file)

Reply:
this PR has nothing to do with implicitly enabling.. the dokku codebase implicitly enables; this pr just allows to disable it.. so if it is disabled it is disabled; if it is enabled you get the same message